final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,267 @@
+import requests
+import argparse
+import json
+import logging
+import socket
+import bs4
+from bs4 import BeautifulSoup
+import CustomException
+
+
+log = logging.getLogger("Log")
```

Comment:
It is better to avoid using global variables, but if you cannot help it, then they should be declared in uppercase, e.g. `VERSOIN`